pages/en/contrib/LTS.md

```diff
@@ -45,9 +44,12 @@ Changes are limited to:
  Semver-minor changes are only permitted if required for bug fixes. Semver-major
  changes are only permitted if required for critical security and bug fixes.
 
-## Maintenance versions
+## Maintenance LTS
 
-When a new major version (for example, 4.0) is released, the oldest LTS version (for example, 2.x) enters maintenance mode, where it will stay at least for another six months and so long as the Node.js LTS versions available at release time are maintained by the Node.js project.
+When a new major version (for example, 4.0) is released, the oldest Active LTS
+version (for example, 2.x) enters Maintenance LTS mode, where it will stay for
+so long as the Node.js LTS versions available at release time are maintained by
+the Node.js project, but at least for six months.
```

Comment:
but at least for 6 months may possibly end up conflicting with Module LTS Policy :/ -- Which is what I think we should adhere for Maintenance LTS.

Reply:
I guess the concern / my point of confusion comes from the following: 

LB 2.x is supported till Node 8.x (December 2019) **OR** 6 months after 4.0 release. If we release in October -- so in January do we support LB 2.x on Node 10.x? Do we even know if everything will work fine on Node 10.x with LB 2.x (unlikely but possible)? (This is the source of my concern ...). 

I think a way around my confusion will be to have a note that no version will ever support a version of Node that is no longer supported by Node.js.